CHANGELOG.md

```diff
@@ -35,6 +35,29 @@
   });
   ```
 
+- Adds feedback form ([#4320](https://github.com/getsentry/sentry-react-native/pull/4328))
+
+  You can add the form component in your UI and customise it like:
+  ```jsx
+  import { FeedbackFormScreen } from "@sentry/react-native";
+  ...
+  <FeedbackFormScreen
```

Comment:
Let's just how the min config in the change log, so ideally with no props. (If we can do that, see my other comments.)

Reply:
Updated the changelog with 4b290a219787b50e5cc55c220b4fc25a98ee1dd5